app/upgrades/v4/upgrade.go

```diff
@@ -113,51 +100,6 @@ func migrateModuleParams(ctx sdk.Context, keepers *keepers.AppKeepers) {
 	baseapp.MigrateParams(ctx, baseAppLegacySS, &keepers.ConsensusParamsKeeper)
 }
 
-// LoadDeprecatedParamsSubspaces loads the deprecated param subspaces for each module
-// used to support the migration from x/params to each module's own store
-func LoadDeprecatedParamsSubspaces(keepers *keepers.AppKeepers) {
-	for _, subspace := range keepers.ParamsKeeper.GetSubspaces() {
-		var keyTable paramstypes.KeyTable
-		switch subspace.Name() {
-		// Cosmos SDK modules
-		case authtypes.ModuleName:
-			keyTable = authtypes.ParamKeyTable()
-		case banktypes.ModuleName:
-			keyTable = banktypes.ParamKeyTable()
-		case stakingtypes.ModuleName:
-			keyTable = stakingtypes.ParamKeyTable()
-		case minttypes.ModuleName:
-			keyTable = minttypes.ParamKeyTable()
-		case distrtypes.ModuleName:
-			keyTable = distrtypes.ParamKeyTable()
-		case slashingtypes.ModuleName:
-			keyTable = slashingtypes.ParamKeyTable()
-		case govtypes.ModuleName:
-			keyTable = govv1.ParamKeyTable()
-		case crisistypes.ModuleName:
-			keyTable = crisistypes.ParamKeyTable()
-
-		// Dymension modules
-		case rollapptypes.ModuleName:
-			keyTable = rollapptypes.ParamKeyTable()
-		case sequencertypes.ModuleName:
-			keyTable = sequencertypes.ParamKeyTable()
-
-		// Ethermint  modules
-		case evmtypes.ModuleName:
-			keyTable = evmtypes.ParamKeyTable()
-		case feemarkettypes.ModuleName:
-			keyTable = feemarkettypes.ParamKeyTable()
-		default:
-			continue
-		}
-
-		if !subspace.HasKeyTable() {
-			subspace.WithKeyTable(keyTable)
-		}
```

Comment:
LoadDeprecatedParamsSubspaces is useless and unnecessary
it does nothing because the adjusted subspace isn't passed anywhere
The only place that needs it is the consensus params upgrade but that does it by itself

Reply:
outdated comment I believe